data/commodities.txt

```diff
@@ -891,3 +891,21 @@ trade
 		"surface-to-air missiles"
 		"targeting systems"
 		"tents"
+	# This list is mostly a placeholder. It should be replaced with one that better fits Syndicate lore.
+	commodity "Syndicate-banned items"
+		"religious texts"
+		"religious items"
+		"religious paraphernalia"
+		"union propaganda"
+		"women's rights propaganda"
+		"worker's rights propaganda"
+		"anti-Syndicate propaganda"
+		"Marxist texts"
+		"open-source software"
+		"replicas of the Fearless Girl statue"
```

Comment:
Actually, there were many people that were offended by the Fearless Girl since it caused a common misconception of the Wall Street Bull. The Bull represents symbol of financial optimism and prosperity, including of the common people. The Fearless Girl twisted the meaning to be oppression by the rich. You can argue, though, that the "financial optimism and prosperity" of Wall Street mostly goes to the rich. In that case, the Fearless Girl had the intended meaning, in a roundabout sort of way.

So, yes, there are many people that are legitimately offended by the Fearless Girl statue, just not for the reason that most would anticipate. That's one of the reasons the statue was moved, and it is one of the reasons it was allowed to stay for so long.

Reply:
Actually, this one is so confusing that I'll just take it out.